packages/bridge/src/runtime/app.plugin.mjs

```diff
@@ -58,16 +59,17 @@ export default async (ctx, inject) => {
   nuxtApp.callHook = nuxtApp.hooks.callHook
 
   const middleware = await import('#build/middleware').then(r => r.default)
+
   nuxtApp._middleware = nuxtApp._middleware || {
-    global: [],
+    global: globalMiddleware,
     named: middleware
   }
 
   ctx.app.router.beforeEach(async (to, from, next) => {
     nuxtApp._processingMiddleware = true
 
     for (const middleware of nuxtApp._middleware.global) {
-      const result = await callWithNuxt(nuxtApp, middleware, [to, from])
+      const result = await middleware(ctx)
```

Comment:
shouldn't we still call it with NuxtApp?

Reply:
The global middleware expects `defineNuxtRouteMiddleware`.
Because `defineNuxtRouteMiddleware` wraps nuxt 2 middleware, I don't think it is needed to use NuxtApp.

https://github.com/nuxt/bridge/blob/913b8e5498d5c4f95637da3a27ad095e30be3578/packages/bridge/src/runtime/composables.ts#L166-L180

Reply:
nice 👌